nbs/example.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Same comment: re <code>annotations</code></p><p> </p><p>Also this might be a good opportunity to mention <code>nbdev_help</code> , which allows you to see all the commands </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>